2/README.md

```diff
@@ -3,80 +3,178 @@ domain: rfc.tango-controls.org
 
-* Be usable as ...
+* Be usable as a logical abstraction of hardware. For example the same GUI application may be used to control power 
+  supplies powering magnets and powering heat coils.
 
-* Be compatible ...
+* Expose, to Users and other Actors of the Tango Controls System, a semantic interface which is well understood, 
+  intuitive, and match the object-oriented paradigm. For example, a power supply Device may be implemented in Java, 
+  Python or C++ language. Irrespectively to the implementation language, Attributes *current* and *voltage* can be 
+  implemented as field members of an object, while *on* and *off* commands can be implemented as methods of the 
+  same object.     
+  
 
 ### Use Cases
 
-There are X main use cases for Template:
-
-* To reduce the entry cost of RFC submission.
+Typical use cases for Device are:
 
-* To make coherent the RFC
+* It is common for particles accelerator systems to use a set of power supplies for powering a magnet system. 
+  Power supplies may come from different vendors. These power supplies may use different communication interfaces
+  and protocols for a remote control. However, there is a need to steer these power supplies from a remote locations in 
+  a unified and consistent way. Steering includes setting and reading of voltages and currents as well as switching the 
+  power supplies on and off, each individually. Having them integrated into a system as Devices, each with an unique name, 
+  with Current and Voltage attributes and On() and Off() commands fulfills the requirement and assure consistency.
 
-...
+* Remote monitoring of a complex device like aa electro-magnet, which is powered by power supplies and requires a cooling
+  system may be provided by a Device with the following attributes: MagneticField, CoilTemperature, State and Status and 
+  with the commands On() and Off(). This Device, to serve its interface is using information coming (directly or through
+  other devices) from power supplies and cooling systems.  
 
 ## Specification
 
 A Tango Device is a strict definition of a distributed object. 
-* configuration are represented in the form of properties.
-* data are represented in the form of attributes.
-* actions are represented in the form of commands.
+* configuration is represented in the form of properties.
+* data are represented in the form of Attributes and Pipes.
+* actions are represented in the form of Commands.
+
+Its model can be represented as a defined tree which each elements are from a defined types: Class, Device, Property, 
+Attribute, Pipe, Command following the rules below:
 
-Its model can be represented as a defined tree which each elements are from a defined types: Class, Device, Property, Attribute, Command following the rules below:
+* The Device is a distributed object which SHALL be accessible locally or via network.
+* The Device SHALL be an instance of one Device Class, see [RFC-9]()
```

Comment:
@hardion, probably we should remove references to RFC-9, as all about Device Class is in this document. What do you think?

Reply:
I did remove references to RFC-9